Instruments/source/SyncTreeProducer.cxx

```diff
@@ -164,29 +164,36 @@ class SyncTreeProducer {
             boost::optional<EventInfoBase> event_info_base = CreateEventInfo(event,signalObjectSelector,&summaryInfo,run_period,jet_ordering);
             if(!event_info_base.is_initialized()) continue;
             if(syncMode == SyncMode::HH && !event_info_base->HasBjetPair()) continue;
-            if(syncMode == SyncMode::HH && !event_info_base->GetTriggerResults().AnyAcceptAndMatch(triggerPaths.at(channel))) continue;
+
+            //if(syncMode == SyncMode::HH && !event_info_base->GetTriggerResults().AnyAcceptAndMatch(triggerPaths.at(channel))) continue;
             if(syncMode == SyncMode::HTT && !event_info_base->GetTriggerResults().AnyAccept(triggerPaths.at(channel))) continue;
+            if(syncMode == SyncMode::HH && !event_info_base->GetTriggerResults().AnyAcceptAndMatchEx(triggerPaths.at(channel), event_info_base->GetFirstLeg().GetMomentum().pt(),
+                                                                                                event_info_base->GetSecondLeg().GetMomentum().pt())) continue;
 
 
-            /*
-            static const std::vector<std::string> trigger_patterns = {
+            static const std::vector<std::string> trigger_patterns_vbf = {
                 "HLT_VBF_DoubleLooseChargedIsoPFTau20_Trk1_eta2p1_Reg_v"
             };
             analysis::EventInfoBase::JetCollection jets_vbf;
-            analysis::EventInfoBase::JetPair vbf_jet_pair;
-            jets_vbf = event_info->SelectJets(30, 5, std::numeric_limits<double>::lowest(),analysis::JetOrdering::Pt,
-                                              event.GetSelectedBjetIndicesSet());
-            vbf_jet_pair = event_info->SelectVBFJetPair(jets_vbf);
-            if(vbf_jet_pair.first >= (*event_info)->jets_p4.size()
-                    || vbf_jet_pair.second >= (*event_info)->jets_p4.size())
-                continue;
-            std::vector<ULong64_t> jet_trigger_match = {
-                (*event_info)->jets_triggerFilterMatch.at(vbf_jet_pair.first),
-                (*event_info)->jets_triggerFilterMatch.at(vbf_jet_pair.second)
+            //analysis::EventInfoBase::JetPair vbf_jet_pair;
+            jets_vbf = event_info_base->SelectJets(30, 5, false, false,analysis::JetOrdering::Pt,
```

Comment:
vbf jets from EventInfo should be used, and not the result of event_info_base->SelectJets